stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -696,6 +927,185 @@ void VhloBytecodeInterface::write(TypeExtensionsAttr attr,
   writer.writeSignedVarInts(attr.getBounds());
 }
 
+//===----------------------------------------------------------------------===//
+// Forked Attributes
+//===----------------------------------------------------------------------===//
+
+namespace {
+template <typename AttrOrType>
+bool assertFromVhlo(AttrOrType val) {
+  if (val.getDialect().getNamespace() != "vhlo") {
```

Comment:
`vhlo::VhloDialect::getDialectNamespace()` instead of `"vhlo"`? (Here and in a few other places).